dist/robots.cfg

```diff
@@ -216,7 +238,12 @@ RobotHost <<EOR
     *.lycos.com,
     *.pa-x.dec.com,
     *.phx.gbl,
+    *.pinterest.com,
     *.search.live.com,
+    *.apple.com,
+    *.petalsearch.com,
+    *.bytedance.com,
+    webmaster.petalsearch.com,
```

Comment:
`webmaster.petalsearch.com` shouldn't be needed because it is part of `*.petalsearch.com`, right?

Reply:
correct.. changed.